tests/cobol85/nistrun.src/run_SM.at

```diff
@@ -0,0 +1,313 @@
+SM204A.SUB: in paragraph 'COPY-WRITE-5':
+SM204A.SUB:378: warning: GO TO SECTION 'CCVS-EXIT'
+SM204A.SUB: in paragraph 'BAD-FILE':
+SM204A.SUB:387: warning: GO TO SECTION 'CCVS-EXIT'
+SM204A.SUB: in section 'CCVS-EXIT':
+SM204A.SUB: in paragraph 'CCVS-999999':
+SM204A.SUB:390: warning: GO TO paragraph 'CLOSE-FILES' which is defined in another SECTION
+SM204A.SUB:203: note: 'CLOSE-FILES IN CCVS1' defined here
+])
+AT_CHECK([./SM204A], [0], [], [])
+
+AT_CLEANUP
+
+
+AT_SETUP([SM205A])
+# promoted on 2023-02-04T15:44
+AT_KEYWORDS([SM])
+AT_CHECK([cp -f ${abs_builddir}/cobol85/SM/SM205A.CBL .], [0], [], [])
+AT_CHECK([$COMPILE -std=cobol85 -I ${abs_builddir}/cobol85/copy SM205A.CBL], [0], [], [/home/lefessan/GIT/COBOL/gnucobol/_build/tests/cobol85/copy/K5SDB:1: warning: DATA RECORDS is obsolete in COBOL 85
+SM205A.CBL:81: warning: LABEL RECORDS is obsolete in COBOL 85
+SM205A.CBL:82: warning: DATA RECORDS is obsolete in COBOL 85
+SM205A.CBL: in section 'SORT-INIT':
+SM205A.CBL: in paragraph 'I-2':
+SM205A.CBL:361: warning: GO TO paragraph 'CLOSE-FILES' which is defined in another SECTION
+SM205A.CBL:246: note: 'CLOSE-FILES IN CCVS1' defined here
+SM205A.CBL: in section 'OUTP2':
+SM205A.CBL: in paragraph 'COPY-WRITE-9':
+SM205A.CBL:545: warning: GO TO paragraph 'LIB2E-EXIT' which is defined in another SECTION
+SM205A.CBL:613: note: 'LIB2E-EXIT IN OUTP3' defined here
+])
+AT_CHECK([./SM205A], [0], [], [])
+
+AT_CLEANUP
+
+AT_SETUP([SM206A])
+# promoted on 2023-02-04T15:44
+AT_KEYWORDS([SM])
+AT_CHECK([cp -f ${abs_builddir}/cobol85/SM/SM206A.CBL .], [0], [], [])
+AT_CHECK([$COMPILE -std=cobol85 -I ${abs_builddir}/cobol85/copy -x -fdebugging-line -debug SM206A.CBL], [0], [], [SM206A.CBL: in section 'SECT-SM206-0002':
+SM206A.CBL: in paragraph 'PST-WRITE-006':
+SM206A.CBL:550: warning: value size exceeds data size
+SM206A.CBL:550: note: value size is 21
+SM206A.CBL:76: note: 'FEATURE' defined here as PIC X(20)
+SM206A.CBL: in paragraph 'PST-DELETE-11':
+SM206A.CBL:667: warning: GO TO SECTION 'CCVS-EXIT'
+SM206A.CBL: in section 'CCVS-EXIT':
+SM206A.CBL: in paragraph 'CCVS-999999':
+SM206A.CBL:709: warning: GO TO paragraph 'CLOSE-FILES' which is defined in another SECTION
+SM206A.CBL:223: note: 'CLOSE-FILES IN CCVS1' defined here
+])
+AT_CHECK([./SM206A], [0], [], [])
+
+AT_CLEANUP
+
+AT_SETUP([SM207A])
+# promoted on 2023-02-04T15:44
+AT_KEYWORDS([SM])
+AT_CHECK([cp -f ${abs_builddir}/cobol85/SM/SM207A.CBL .], [0], [], [])
+AT_CHECK([$COMPILE -std=cobol85 -I ${abs_builddir}/cobol85/copy SM207A.CBL], [0], [], [SM207A.CBL: in section 'CCVS-EXIT':
+SM207A.CBL: in paragraph 'CCVS-999999':
+SM207A.CBL:360: warning: GO TO paragraph 'CLOSE-FILES' which is defined in another SECTION
+SM207A.CBL:202: note: 'CLOSE-FILES IN CCVS1' defined here
+])
+AT_CHECK([./SM207A], [0], [], [])
+
+AT_CLEANUP
+
+AT_SETUP([SM208A])
+# promoted on 2023-02-04T15:44
+AT_KEYWORDS([SM])
+AT_CHECK([cp -f ${abs_builddir}/cobol85/SM/SM208A.CBL .], [0], [], [])
```

Comment:
Better:

```diff
-AT_CHECK([cp -f ${abs_builddir}/cobol85/SM/SM208A.CBL .], [0], [], [])
-AT_CHECK([$COMPILE -std=cobol85 -I ${abs_builddir}/cobol85/copy SM208A.CBL], [0], [], [SM208A.CBL: in section 'SECT-SM208A-001':
+AT_CHECK([$COMPILE -std=cobol85 -Wno-goto-different-section -I ${abs_builddir}/cobol85/copy ${abs_builddir}/cobol85/SM/SM208A.CBL], [0], [], [])
```

But I'd actually suggest to at least define an additional `COMPILE85` as `$COMPILE -std=cobol85 -I ${abs_builddir}/cobol85/copy` - or even a macro so one could use

`NIST_COMPILE([SM208A])`